### PR TITLE
Updated to the latest flow library

### PR DESCRIPTION
**Pull request by levb**
`f.State`->`f.GetState()`

### CODE REVIEW

**Review by hanzei — APPROVED** [review]
LGTM

File: server/plugin/flows.go [hunk 1]
```diff
@@ -335,15 +336,15 @@ func (fm *FlowManager) submitDelegateSelection(f *flow.Flow, submitted map[strin
...
 	return stepDelegateConfirmation, flow.State{
-		"Delegated": delegate.GetDisplayName(model.ShowNicknameFullName),
+		keyDelegatedTo: delegate.Username,
 	}, nil, nil
 }
 
 func (fm *FlowManager) stepDelegateConfirmation() flow.Step {
 	return flow.NewStep(stepDelegateConfirmation).
-		WithText("GitHub integration setup details have been sent to @{{.Delegated}}").
+		WithText("GitHub integration setup details have been sent to @{{ .DelgatedTo }}").
```
Comment (catalintomai) [inline server/plugin/flows.go:345]: DelgatedTo -> DelegatedTo

**Comment by DHaussermann** [comment]
Tested and passed

Regression tested the setup flow. Ensured everything is still functional including on HA.
No issues found or error seen regarding flow state.  
LGTM!

Thanks @levb for this fix. Please merge if dev review comment has been addressed.

**Review by DHaussermann — APPROVED** [review]

**Pull request merged**